extmod/uasyncio/lock.py

```diff
@@ -19,7 +19,7 @@ def locked(self):
 
     def release(self):
         if self.state != 1:
-            raise RuntimeError
+            raise RuntimeError("Lock is not acquired.")
```

Comment:
I guess this change is for usability, instead of a blank `RuntimeError` it gives a proper message.  But is it necessary to have the exact same message as CPython?  Usually MicroPython's error messages are shorter versions of their CPython counterpart.  So this here could be made `"Lock not acquired"`.  Every byte counts!

Reply:
I guess so. I can't imagine anyone comparing that RuntimeError message for equality to "Lock is not acquired.". That would be strange..